index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+</head>
+
+<body>
+
+    <header class="navibar">
+        <p class="name">John Doe</p>
+        <img src="Images/hamburger.png" class="hamburger" alt="menu">
+    </header>
+    <div class="main">
+        <div class="submain">
+            <h1 class="introduction">Hey There.<br> I’m Alex</h1>
+            <h2 class="occupation">I’am a Software Developer</h2>
+            <p class="services">I can help you build a product , feature or website Look through some of my work and
+                experience!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+            <ul class="icons">
+                <li>
+                    <i class="fab fa-github"></i>
+                    <i class="fab fa-linkedin-in"></i>
+                    <i class="fab fa-angellist"></i>
+                    <i class="fab fa-twitter"></i>
+                    <i class="fab fa-medium"></i>
+                </li>
+            </ul> 
```

Comment:
- [ ] Instead of wrapping all your icons inside a single `li`, it would be nice to use a different `li` for each icon, like: 
```
<li><i class="fab fa-github"></i>
<li><i class="fab fa-linkedin"></i>
```
So this would keep your code more semantic and readable, so please make sure to apply this change.